modules/network-observability-total-resource-usage.adoc

```diff
@@ -21,26 +20,16 @@ Metrics exported to Prometheus can impact the resource usage. Cardinality values
 .Total average resource usage
 [%autowidth, options="header"]
 |===
-| Sampling value | Parameters | Test 1 (25 nodes) | Test 2 (65 nodes) | Test 3 (120 nodes)
-.6+| *Sampling = 1* | *With Loki* 3+|
-| Total NetObserv CPU Usage | 3.24 | 3.42 | 7.30
-| Total NetObserv RSS (Memory) Usage | 14.09 GB | 22.56 GB | 36.46 GB
-| *Without Loki* 3+|
-| Total NetObserv CPU Usage | 2.40 | 2.43 | 5.59
-| Total NetObserv RSS (Memory) Usage | 6.85 GB | 10.39 GB | 13.92 GB
-.6+| *Sampling = 50* | *With Loki* 3+|
-| Total NetObserv CPU Usage | 2.04 | 2.36 | 3.31
-| Total NetObserv RSS (Memory) Usage | 8.79 GB | 19.14 GB | 21.07 GB
-| *Without Loki* 3+|
-| Total NetObserv CPU Usage | 1.55 | 1.64 | 2.70
-| Total NetObserv RSS (Memory) Usage | 6.71 GB | 10.15 GB | 14.82 GB
-.6+| *Sampling = 400* | *With Loki* 3+|
-| Total NetObserv CPU Usage | 1.71 | 1.44 | 2.36
-| Total NetObserv RSS (Memory) Usage | 8.21 GB | 16.02 GB | 17.44 GB
-| *Without Loki* 3+|
-| Total NetObserv CPU Usage | 1.31 | 1.06 | 1.83
-| Total NetObserv RSS (Memory) Usage | 7.01 GB | 10.70 GB | 13.26 GB
-|===
+| Sampling value | Resources used | Test 1 (25 nodes) | Test 2 (250 nodes)
```

Comment:
the table is screwed up, there's something wrong somewhere :-) 
![image](https://github.com/user-attachments/assets/117e1be9-a99f-4f55-87d0-6cf0f20b43c2)

Reply:
Oooo sorry, when I added the CPU per node, I forgot to adjust the row spanning 🥴 🤦‍♀️ Its fixed now:
![image](https://github.com/user-attachments/assets/ef247eb9-9b20-463f-86c3-395997dcfaa3)